packages/examples/examples/rollup/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "rollup-snap",
+  "version": "0.0.0",
+  "private": true,
+  "description": "An example Snap built using TypeScript and Rollup",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MetaMask/snaps-skunkworks.git"
+  },
+  "license": "ISC",
+  "main": "src/index.js",
+  "files": [
+    "dist/",
+    "snap.manifest.json"
+  ],
+  "scripts": {
+    "build": "rollup --config",
+    "manifest": "mm-snap manifest --fix",
+    "eval": "mm-snap eval -b dist/snap.js",
+    "serve": "concurrently \"rollup --config --watch\" \"mm-snap serve\""
+  },
+  "devDependencies": {
+    "@babel/core": "^7.17.12",
+    "@babel/preset-typescript": "^7.17.12",
+    "@metamask/rollup-plugin-snaps": "^0.14.0",
+    "@metamask/snap-types": "^0.14.0",
+    "@metamask/snaps-cli": "^0.14.0",
+    "@rollup/plugin-babel": "^5.3.1",
+    "@types/node": "^17.0.33",
+    "concurrently": "^7.1.0",
+    "rollup": "^2.73.0",
+    "rollup-plugin-execute": "^1.1.1",
+    "tslib": "^2.4.0",
+    "typescript": "^4.6.4"
+  },
+  "engines": {
+    "node": ">=12.11.0"
```

Comment:
Should we match the other packages with `"node": ">=14.0.0"`? I may have missed this for other example snaps.